src/search.py

```diff
@@ -8,15 +8,23 @@
 
 
 class SearchView(MethodView):
-    RESULT_LIMIT = 10
+    DEFAULT_RESULT_LIMIT = 10
+    MAX_RESULT_LIMIT = 100
 
     @auth.login_required
     def get(self):
         try:
             name = flask.request.args.get('name')
             url = flask.request.args.get('url')
+            result_limit = flask.request.args.get('limit')
             if name is None and url is None:
                 raise ValueError('One of params "name", "url" is required.')
+            if result_limit is None:
+                result_limit = DEFAULT_RESULT_LIMIT
+            else:
+                result_limit = int(result_limit)
```

Comment:
simplification: `min(result_limit, MAX_RESULT_LIMIT)`

Reply:
or using the default param from comment above